website/users.php

```diff
@@ -285,11 +285,30 @@
 			$token = $users->createPasswordResetToken($user['userid']);
 ?>
 		<div style="border: 1px solid #DDAA88; padding: 0 1em; margin-bottom: 1em">
+		<script type="text/javascript">
+		// partially stolen from js/client.js
+		function copyResetUrl() {
+			var appended = $('.pfx-topbar')[0];
+			var dummyLink = document.createElement("input");
+			// You can only "select" an input field, so we have to do this dumb hack of faking one.
+			dummyLink.id = "dummyLink";
+			dummyLink.value = $("#token").text();
+			dummyLink.style.position = 'absolute';
+			appended.appendChild(dummyLink);
+			dummyLink.select();
+			document.execCommand("copy");
+			appended.removeChild(dummyLink);
+			$('#copy').text('Copied!');
+		}
+		</script>
```

Comment:
This doesn't work in searches. JS needs to go in `website/js/ladder.js`.

Reply:
Done.